lib/routing.js

```diff
@@ -59,27 +59,25 @@ module.exports = [
   }
 ];
 
-if (config.get('env') !== 'prod') {
-  module.exports.push.apply(module.exports, [
-    {
-      method: 'GET',
-      path: v('/clients'),
-      config: require('./routes/client/list')
-    },
-    {
-      method: 'POST',
-      path: v('/client'),
-      config: require('./routes/client/register')
-    },
-    {
-      method: 'POST',
-      path: v('/client/{client_id}'),
-      config: require('./routes/client/update')
-    },
-    {
-      method: 'DELETE',
-      path: v('/client/{client_id}'),
-      config: require('./routes/client/delete')
-    }
-  ]);
-}
+exports.clients = [
```

Comment:
naming nit: this is a list of routes, so it would be clearer as something like "clientRoutes" or "internalRoutes".